db/migrate/20100929044155_add_persistence_token_to_users_and_admins.rb

```diff
@@ -1,13 +0,0 @@
-class AddPersistenceTokenToUsersAndAdmins < ActiveRecord::Migration
```

Comment:
This is because when schema.rb was updated I should have cleaned out the migrations and I did not. Because of that I could never get the tests to work. I could have done that in another pull request but given this is due to be merged and that the clean up has to be done before any thing with migrations passes the test this was the easiest way of getting the tests to pass. Other wise I would have this pull request which is needed for two others to consider working. Which needed another pull request before its tests could work.

I tried to keep it simple :(